shub/config.py

```diff
@@ -16,6 +16,8 @@
                         pwd_hg_version, pwd_git_version, pwd_version)
 
 
+SH_IMAGES_PREFIX = 'images.scrapinghub.com/project/'
+SH_IMAGES_REGISTRY = SH_IMAGES_PREFIX + '{project}'
```

Comment:
That's a good idea to validate repository if `images.sh.com` is used, I had a couple of similar issues in the past. Don't think there's such use case that could be wanted behaviour - that looks like a hidden bug, and can lead to pretty tricky cases. Also if image is already in our registry, pushing the same image with other tag is almost instant operation, as it will reuse all the layers already existing in the registry.